content/rc/administration/security/_index.md

```diff
@@ -28,18 +29,22 @@ You may change your subscription at any time by going to Databases -> Configurat
 
 ## Securing Connection to Your Database with SSL/TLS
 
-You can [secure your database connections]({{< relref "/rc/securing-redis-cloud-connections.md" >}}) with SSL/TLS.
+You can [secure your database connections]({{< relref "/rc/administration/security/securing-redis-cloud-connections.md" >}}) with SSL/TLS.
 
-## Redis Password
+## Default User
 
-We recommend that you add a password for your database.
-This password is only for access to the data in the database, and not for database or cluster administration.
+When you [enable the default user]({{< relref "/rc/administration/setup/create-database.md" >}}) for your database,
+all connections to the database must use the [AUTH command](https://redis.io/commands/auth) to authenticate with the default user password.
```

Comment:
In the UI it is "default user password" in other words the password for the default user. I avoid possessives like the plague. But I see how this can be confusing.

Reply:
Possessives are a feature of the language. If they clarify, it's good to use them.

Reply:
The default user for your database, which is enabled by default,
requires all connections to the database to use the [AUTH command](https://redis.io/commands/auth) to authenticate with the default user password.